src/utils/utils.ts

```diff
@@ -73,3 +87,31 @@ export const calculateTimeRemaining = (startDate: Date) => {
     seconds: seconds % 60,
   };
 };
+
+/**
+ * Returns whether the current date is in the range of a given start and end date.
+ *
+ * @param startDate - A date, must be before [endDate].
+ * @param endDate - A date, must be after [startDate].
+ * @returns A boolean stating if the current date is greater than or equal to startDate and less than or equal to endDate.
+ */
+export const dateInRange = (date: Date, startDate: Date, endDate: Date) => {
```

Comment:
You have three parameters here but only have two documented. You also mentioned "current date" so I assumed that you would have a default value no? I would also rename the `date` parameter to something more clear.